tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -36,8 +36,8 @@ Define {
 	# override DMIC default definitions
 	DMIC0_HOST_PIPELINE_ID		13
 	DMIC0_DAI_PIPELINE_ID		14
-	DMIC0_HOST_PIPELINE_SINK	'copier.host.13.1'
-	DMIC0_DAI_PIPELINE_SRC		'copier.DMIC.14.1'
+	DMIC0_HOST_PIPELINE_SINK	'copier.host.10.capture'
+	DMIC0_DAI_PIPELINE_SRC		'copier.DMIC.0.capture'
```

Comment:
do we need to add the pipeline ID? It's used for topology1 and helps visualize the pipeline boundaries.

The position in the pipeline could also be useful for controls.

Maybe we need 
````
<pipeline> <position in pipeline> 
````
for all modules, and 
````
<dai_index>
```` 
in addition as a suffix for copiers only? (with the caveat on dai_id for ALH above)